torch_xla/experimental/pallas_kernels/quantized_matmul_kernel.py

```diff
@@ -216,6 +205,29 @@ def quantized_matmul_int8(
   # For best performance, only enable this behavior when single input block is used per batch.
   save_q_x = quantize_activation and n_in == 1 and n_out > 1
 
+  acc_dtype = jnp.int32 if quantize_activation else jnp.float32
+
+  # Calculate in/out VMEM size
+  vmem_in_out = (
```

Comment:
you need to "*2" because of double buffering

Reply:
Modified the code to account for double buffering.